src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
-body {
-  margin: 0;
+* {
+  box-sizing: border-box;
+
+  $main-accent: #060B35;
+  $secondary: #616070;
+  $blue-accent: #00ACDC;
+  $white-accent: #fff;
+
+  body {
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-accent;
+    font-family: Roboto, sans-serif;
+    font-weight: 400;
+
+    .card-block {
+      width: 200px;
+      min-height: 408px;
+      padding: 32px 16px 16px;
+      border: 1px solid #F3F3F3;
+      border-radius: 5px;
+      display: flex;
+      flex-direction: column;
+      position: relative;
+
+      .card-img {
+        width: 160px;
+        height: 134px;
+        margin-left: 3px;
+      }
+      .card-title {
+        margin-top: 40px;
+        margin-bottom: 4px;
+        font-weight: 500;
+        font-size: 12px;
+      }
+
+      .card-subtitle {
+        color: $secondary;
+        font-size: 10px;
+        line-height: 14px;
+        margin-bottom: 17px;
+      }
+
+      .group-one {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        .stars {
```

Comment:
it must placed in separate file as a BEM block and need to copy all modificators not just for 4 active stars